tools/c7n_azure/c7n_azure/resources/front_door.py

```diff
@@ -32,3 +33,32 @@ class resource_type(ArmResourceManager.resource_type):
             'resourceGroup'
         )
         resource_type = 'Microsoft.Network/frontDoors'
+
+@FrontDoor.filter_registry.register('waf-not-enabled')
```

Comment:
general preference for just doing waf-enabled with a state param to detect on either enabled or not enabled, else things read super odd, when wrapping with a not block